x-pack/plugins/transform/public/app/sections/transform_management/components/action_discover/use_action_discover.tsx

```diff
@@ -27,8 +27,7 @@ export type DiscoverAction = ReturnType<typeof useDiscoverAction>;
 export const useDiscoverAction = (forceDisable: boolean) => {
   const appDeps = useAppDependencies();
   const { share } = appDeps;
-  const savedObjectsClient = appDeps.savedObjects.client;
-  const dataViews = appDeps.data.dataViews;
+  const dataViewsContract = appDeps.data.dataViews;
```

Comment:
nit, `data` and `application` could be added to the line above along with `share`, to clean this code up a bit.
In fact they could all be added to line 28 and `appDeps` removed

Reply:
Done in https://github.com/elastic/kibana/pull/131819/commits/36d639885f4609acb2982b083b9639e7a47b7956